cloudpickle/cloudpickle.py

```diff
@@ -69,6 +69,43 @@
     from io import BytesIO as StringIO
     PY3 = True
 
+
+try:
+    from ctypes import pythonapi, py_object, c_int, PYFUNCTYPE
+except ImportError:
+    supports_recursive_closure = False
+
+    def compress_closure(closure):
+        return closure
+
+    def decompress_closure(compressed_closure):
+        return compressed_closure
+
+    def fill_cells(cells, values):
+        pass
+else:
+    supports_recursive_closure = True
+
+    def compress_closure(closure):
```

Comment:
The point here is that the closure "compression" is different on different versions of Python. on pypy this is an identity function. I didn't want to leak the implementation here, the only important thing is that this works when `decompress_closure` is called on the result.